content/docs/command-reference/init.md

```diff
@@ -100,6 +100,54 @@ DVC sets the `core.no_scm` config option value to `true` in the DVC
 that even if the project is tracked by Git, or if Git is initialized in it
 later, DVC will keep operating detached from Git in this project.
 
+### Nested DVC projects and repositories
```

Comment:
Doing PR this way makes it hard to review (we lost the context since we removed the whole section somewhere else). :(

Reply:
Let's revert the way it was before please and then do changes on top?

Reply:
We already reverted all of the changes about this in the previous PR.

Reply:
OK I think you mean having the code blocks back in the --subdir section... Hold on ⌛

Reply:
Reverted for now but those changes were already merged to master so reverting here looks like a change. This is still a draft BTW, not quite ready for review.

Reply:
I can remove the new section here, and then we can merge this, so a following PR shows you the changes they way I think you'd like to see them ⌛

Reply:
Ready.